README.md

```diff
@@ -2,82 +2,56 @@
-[special formats](http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide).
-
-#### gct format conversion
-Convert a gene expression tab separated values (tsv) file provided
-into a gene cluster text (gct) file for use in GenePattern notebooks.
-In order to create a gct formatted file from a tsv refine.bio data file,
-reference the
-[`create_gct_file.R` script](https://github.com/AlexsLemonade/refinebio-examples/blob/master/scripts/create_gct_file.R), followed by `-f` argument with the name
-of the file in your current directory that you would like to convert.
-Note: This script requires optparse library. If optparse library is not
-installed, this script will install it for you.
-
-##### Example of usage in command line:  
- ```bash
- $ Rscript create_gct_file.R -f GSE111111.tsv -o outputfilename -r
- ```
- Options:   
-`-f` :name of the file in your current directory that you would like to convert.    
-`-r` :file of the same name as the output will be rewritten (*optional*)       
-`-o` :name for the output file (*optional*)       
-
-Optionally you can designate the name of the output file by adding an `-o`
-argument.
-The "gct" suffix will be added if you do not add it yourself.
-Be sure to either have the script and input file in your current working
-directory, or put type out the full directory path for the script and/or input
-file. eg `/users/Bob/Desktop/create_gct_file.R`
-
-#### CLS format file creation
-CLS formatted files tell information regarding the groups or phenotype of the
-different samples and are necessary for doing gene expression differential
-analysis using GenePattern's notebooks. If you've already created a gct format
-file from your data, you can create a a CLS format using GenePattern's online
-[CLSFileCreator](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ClsFileCreator/4)
-
-# Table of Contents  
+Each directory in this repository is a set of example workflows to follow.
+See the list of `Example Workflows` below for list of the analysis topics.
+
+*Each directory is its own set of example workflows and contains:*
+
+- A`README` that introduces you to the concepts and example analyses in that module.
+- Example dataset(s) in the `data/` folder
+- An R markdown (`Rmd`) file that you can use in RStudio to run the analysis
+- An R notebook (`nb.html`) file that is the resulting output of the Rmd file
+but in a prettier format.
+- A `results` and/or `plots` folder(s) that contains the output of the analyses.
+
+As you get more comfortable with the examples, we encourage you to apply these
+example workflows to your own refine.bio data.
+After downloading a refine.bio dataset, you can analyze them in these examples
+by copying and pasting your data and metadata TSV files from your refine.bio
+download file into the respective `data/` folder for that module.
+You'll need to change any directory paths in the notebook to have the name of
```

Comment:
I think `directory paths` here is unclear because we're talking about how we recommend putting the data in the `data/` folder. How about something like `file names` instead?